inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/send/ProxyManager.java

```diff
@@ -49,6 +51,8 @@ public class ProxyManager {
     private String secretId;
     private String secretKey;
     private volatile boolean timerStarted = false;
+    private static final int MAX_RETRY_TIMES = 1440;
+    private static final int RETRY_INTERVAL_MS = 1000;
```

Comment:
Suggest a slightly larger retry interval

Reply:
It has update to 10000 MS